src/Button/styles.ts

```diff
@@ -7,15 +7,15 @@ export const getVariantStyles = (variant: VariantType = 'default', theme?: Theme
       color: 'btn.text',
       backgroundColor: 'btn.bg',
       boxShadow: `${theme?.shadows.btn.shadow}, ${theme?.shadows.btn.insetShadow}`,
-      '&:hover:not([disabled])': {
+      '&:hover:not([disabled]):not([aria-disabled])': {
```

Comment:
The a11y team told us not to pass `aria-disabled` for inactive buttons since they still accept interactions such as focus or click.

However, I chose to keep the styles for `disabled` and `aria-disabled` the same to align with PVC and PCSS.